modules/nw-http2-haproxy.adoc

```diff
@@ -6,31 +6,37 @@
 [id="nw-http2-haproxy_{context}"]
 = Enabling HTTP/2 Ingress connectivity
 
-You can enable transparent end-to-end HTTP/2 connectivity in HAProxy. It allows application owners to make use of HTTP/2 protocol capabilities, including single connection, header compression, binary streams, and more.
+You can enable transparent end-to-end HTTP/2 connectivity in HAProxy, so that an application owner can access HTTP/2 protocol capabilities, which includes single connection, header compression, binary streams, and other features.
 
-You can enable HTTP/2 connectivity for an individual Ingress Controller or for the entire cluster.
+You can either enable HTTP/2 connectivity for an individual Ingress Controller or for the entire cluster.
 
-To enable the use of HTTP/2 for the connection from the client to HAProxy, a route must specify a custom certificate. A route that uses the default certificate cannot use HTTP/2. This restriction is necessary to avoid problems from connection coalescing, where the client re-uses a connection for different routes that use the same certificate.
+To enable the use of HTTP/2 for a connection from the client to a HAProxy service, a route must specify a custom certificate. A route that uses the default certificate cannot use HTTP/2. This restriction is necessary to avoid problems from connection coalescing, where the client re-uses a connection for different routes that use the same certificate.
```

Comment:
We use "an HAProxy" rather than "a HAProxy" elsewhere in the documentation.  It is a stylistic choice that depends on how one pronounces the 'h', but "an HAProxy" is more consistent with existing documentation.

In any case, why write "a(n) HAProxy service" instead of simply HAProxy?  In my opinion, it is clearer to write "HAProxy", "an HAProxy instance", or "a router pod".  The word "service" could imply that a Kubernetes service is involved, which is not always the case.